fe/fe-core/src/main/java/org/apache/doris/load/loadv2/SparkLoadJob.java

```diff
@@ -695,6 +695,8 @@ private void clearJob() {
             // clear job infos that not persist
             sparkLoadAppHandle = null;
             resourceDesc = null;
+            etlOutputPath = "";
+            appId = "";
```

Comment:
Why set empty string here?

Reply:
If the job is cancelled by user, the `clearJob()` will be called twice.
Set thess shtring empty,to ensure the idempotency of the `clearJob` function.